simp_le.py

```diff
@@ -1183,5 +1210,33 @@ def test_error_exit_codes(self, dummy_stderr):
                 assert False
 
 
+class IOPluginTests(TestCase):
+    """Integration tests for plugin interface."""
+
+    # this is unittest suite | pylint: disable=missing-docstring
+
+    def setUp(self):
+        self.root = tempfile.mkdtemp()
+        self.key_path = os.path.join(self.root, 'key.pem')
+        test_key = os.path.join('testdata', "rsa2048_key.pem")
```

Comment:
Please use `gen_pkey`: I'd like all tests to be self-contained within one script (e.g. so that they can be run from binaries created with PyInstaller), so we would have to inline entire key, which doesn't look great. If at some point that becomes of an performance concern, we will think about it then.

BTW if we were to use testdata, we would use `pkg_resources` to load such.